internal/serviceapi/projectsettingsapi/resource_schema.go

```diff
@@ -64,7 +64,7 @@ func ResourceSchema(ctx context.Context) schema.Schema {
 }
 
 type TFModel struct {
-	GroupId                                         types.String `tfsdk:"group_id" autogen:"omitjson"`
```

Comment:
q: Is `apiname:groupId` extension needed here? Or altogether adding `apiname` as part of this PR? 
From changes I see we now apply the alias `groupId: projectId` to the paths and ModelTFName so it is working correctly ([reference](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/3934/files#diff-eafb506e0429411e736781c09f3ca96fe8ca5c8b6818042afce130a5944eee6bL85-L89)).